movie-mlops-project/serving/services/mlflow_service.py

```
@@ -0,0 +1,272 @@
+            self.model_version = model_check['version']
+            
+            logger.info("✅ Production 모델 로드 완료!")
+            logger.info(f"   🏷️ 모델 타입: {type(self.model)}")
+            
+            # XGBoost 모델 정보 확인
+            try:
+                if hasattr(self.model, 'n_features_in_'):
+                    logger.info(f"   📊 입력 특성 수: {self.model.n_features_in_}")
+                if hasattr(self.model, 'get_booster'):
+                    logger.info(f"   🚀 XGBoost 모델 확인됨")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)